x-pack/plugins/apm/public/components/app/TransactionOverview/index.tsx

```diff
@@ -71,6 +79,10 @@ export function TransactionOverviewView({
   );
 
   const { data: transactionListData } = useTransactionList(urlParams);
+  const { data: hasMLJob = false } = useFetcher(
+    () => getHasMLJob({ serviceName, transactionType }),
+    [serviceName, transactionType]
+  );
 
   // filtering by type is currently required
   if (!serviceName || !transactionType) {
```

Comment:
`!serviceName` should no longer be necessary check here

Reply:
Good catch! Combining this with the check above.